src/centralstore/incremental/NodeBlock.cpp

```diff
@@ -221,6 +223,11 @@ NodeBlock* NodeBlock::get(unsigned int blockAddress) {
         node_block_logger.error("Error while reading edge reference data from block " + std::to_string(blockAddress));
     }
 
+    if (!NodeBlock::nodesDB->read(reinterpret_cast<char*>(&edgeRefPID), sizeof(unsigned char))) {
+        node_block_logger.error("Error while reading edge reference partition ID data from block " +
```

Comment:
Don't we have to say which graph ID and which partition ID we are referring to here?

Reply:
Not at this NodeDB class level as mentioned in https://github.com/miyurud/jasminegraph/pull/122#discussion_r606796318

Reply:
Ok